creations/static/stylesheets/radio.sass

```diff
@@ -178,6 +183,23 @@ $mobile-breakpoint: 500px
       @include cleared-spans
       margin-bottom: $whitespace
 
+      .external-links
+        display: flex
+        align-items: center
```

Comment:
I don't _love_ that this is repeated, but don't mind waiting to fix

Reply:
I addressed this by extracting styles for that component to their own sheet. Good idea!